src/scenes/App/components/TermsAndConditions/index.jsx

```diff
@@ -0,0 +1,274 @@
+        <li><FormattedMessage id="tnc.responsibilities4" defaultMessage="We shall have no liability with respect to any damage or loss that was caused due to reliance, of any type, on the information or any other publication or content appearing at the Site and/or Platform, and You are invited to verify the information published at the Site and/or Platform." /></li>
+        <li><FormattedMessage id="tnc.responsibilities5" defaultMessage="We shall not be responsible or liable for any actions or omissions of any internet service provider or any other third party which provides You with access to Site and/or Platform." /></li>
+        <li><FormattedMessage id="tnc.responsibilities6" defaultMessage="We shall not be responsible for any damage or loss You shall incur as a result of modifications, enhancement, termination, suspension or discontinuation of the Site and/or Platform. We will not be responsible for any damage or loss You shall incur as a result of Your use or reliance on the content of any website, mobile site and/or mobile application to which links appear on the Site." /></li>
+        <li><FormattedMessage id="tnc.responsibilities7" defaultMessage="You will indemnify and hold Us harmless against all direct and indirect claims, liabilities, damages, losses, costs and expenses arising from Your breach of these Terms and Conditions." /></li>
+        <li><FormattedMessage id="tnc.responsibilities8" defaultMessage="The Site and Platform are provided “as is”, and We make no warranty or representation, whether express or implied (whether by law, statute, or otherwise), including but not limited to implied warranties and conditions of merchantability, satisfactory quality, fitness for a particular purpose, completeness or accuracy, non-infringement of third parties’ rights or of applicable laws and regulation in respect of the Site and/or Platform, or that the Site and/or Platform will be uninterrupted, timely, secure or error-free, or that defects will be corrected, or will be free of viruses or bugs or as to results or the accuracy of any information through the Site and/or Platform." /></li>
+        <li><FormattedMessage id="tnc.responsibilities9" defaultMessage="You are responsible for maintaining your own wallet. Bodhi is a decentralized application, with no centralized mechanism for accessing your wallet. Bodhi does not keep, store, save, or in any way backup your account details. Therefore Bodhi is unable to assist in any way to recover wallet access. Mismanagement of your wallet details may cause you to lose all your funds that may not be recoverable. We are not responsible for the loss of such funds arising from entering the wrong wallet details." /></li>
+      </ol>
+    </div>
+  );
+
+  renderIp = () => (
```

Comment:
This should be a component, not a method. Make a stateless component please.

Reply:
done